prow/cmd/sub/README.md

```diff
@@ -65,8 +65,12 @@ and value ```prow.k8s.io/pubsub.PeriodicProwJobEvent```, and a payload like so:
     "myLabel":"myValue",
   },
   "annotations":{
+    // GCP project where prowjobs statues are published by prow. Must also provide "prow.k8s.io/pubsub.topic" to take effect.
```

Comment:
Technically support comments, I'd suggest changing L57 to yaml

Reply:
good idea! done